src/main/java/time/TimeInCapitals/showtime/LocalTimeService.java

```diff
@@ -34,31 +36,159 @@ public String getTime() {
 		return localTime;
 	}
 
+	public String getTimeForAfricaCapital(String capital) {
```

Comment:
Bardzo jestem niezadowolony z tego miejsca:

1. Kontynent powinien być przekazywany jako parametr.
2. Na podstawie wartości parametru powinniśmy decydować, jaki algorytm wybieramy.
3. Co się dalej tam dzieje, już nawet nie czytam, widzę tu tylko mnóstwo duplikacji i kodu proszącego się o to, aby go podzielić na mniejsze metody.

Reply:
Omówione, do poprawy po testach.

Reply:
@pablor83 Merguję tego pull requesta, żeby nie wisiał 2 miesiące. Wspomniane wyżej zmiany wrzucisz w nowym.